config/core/test-configs.yaml

```diff
@@ -1271,6 +1271,13 @@ device_types:
     filters:
       - blocklist: {defconfig: ['stm32_defconfig']}
 
+  stm32mp157c-lxa-mc1:
+    mach: stm32
+    class: arm-dtb
+    boot_method: barebox
+    filters:
+      - blocklist: {defconfig: ['stm32_defconfig']}
```

Comment:
Out of interest, why not test this defconfig on this platform?  (I see it's the same for `stm32mp157c-dk2`).

Reply:
I used the dk2 as the template. What is the reason the dk2 is having it in the blocklist?